extension/js/common/core/mime.ts

```diff
@@ -71,6 +71,13 @@ export class Mime {
         decoded.signature = decoded.signature || file.getData().toUtfStr();
       } else if (treatAs === 'publicKey') {
         blocks.push(...Pgp.armor.detectBlocks(file.getData().toUtfStr()).blocks);
+      } else if (treatAs === 'backup') {
+        const backupBlocks = Pgp.armor.detectBlocks(file.getData().toUtfStr()).blocks;
+        // Need to set type to the backup because Pgp detects it as a private key
```

Comment:
Thanks for the comment, I was wondering what is going on. Actually, `privateKey` block and `backup` are the same thing. I didn't realize we already had a block named `privateKey`. I will refactor it a bit after merging.